docs/http/1.basic/http-connection.md

```diff
@@ -0,0 +1,54 @@
+---
+title: TCP 握手和挥手
+sidebar_label: TCP 握手和挥手
+---
+import Img from '../../../src/components/Img'
+
+## 三次握手和四次挥手
+
+### 三次握手
+
+男（客户端-SYN）：喂喂喂，我是你男票，你听的到吗？
```

Comment:
the duplicated content can be removed from 'web'

Reply:
When I sort out related content, I will remove them from the web.